jobs/java-docker-jobs.yml

```diff
@@ -12,6 +12,15 @@
     jobs:
       - '{name}'
 
+- project:
+    name: wsapi
+    description-intro: Builds, tests, & deploys the WSAPI docker image.
+    email-recipients: matt.drees@cru.org
+    base-image: jboss/wildfly:17.0.1.Final
+    jdk: JDK11
```

Comment:
@mikealbert I had Dmitry go ahead and update Wsapi to use Java 11. But I forgot that Jenkins only has Java 7 and 8 installed on it. Also, since Oracle's licensing changes, we need to switch over to openjdk. I'm going to look into https://stackoverflow.com/a/55244659/1174749.

Reply:
actually, I'm going to have wsapi use a maven via docker (maven:3.6.1-jdk-11), from a build.sh file. We'll use app-jobs.yml, and forgo Jenkins' maven/java project support.